types/node/http.d.ts

```diff
@@ -395,11 +395,85 @@ declare module "http" {
         destroy(): void;
     }
 
-    const METHODS: string[];
+    const METHODS: [
```

Comment:
The documentation doesn't specify that these are sorted. This should be defined as `('ACL' | 'BIND' | ...)[]`, not as a tuple.

Reply:
Addressed